spec/lib/js_locale_helper_spec.rb

```diff
@@ -262,7 +262,9 @@ def clear_cache!
   end
 
   describe ".output_client_overrides" do
-    it "should not output deprecated translation overrides" do
+    subject(:client_overrides) { described_class.output_client_overrides("en") }
```

Comment:
Using a `subject` here as it will memoize the result, so when calling this twice, it will just reuse the cached value for the second time.